README.md

```diff
@@ -87,6 +87,34 @@ Then for each game node (typically one per CPU/core):
 PORT={port} SERVER={node-name} node server/gamenode
 ```
 
+### Running and Testing
+
+The game server should be accessible by browsing to localhost:4000. 
+
+You can register 2 or more users, to play against yourself. 
+They can have fake email addresses. 
+You can login as both users either from 2 different browsers, or by 
+using an incognito window. 
+
+There is no concept of an 'admin' user.
+
+If you implement or make changes to a card, you can use manual mode 
+to add it to a deck from within a game. Use manual mode, and the command:
+```
+/add-card <card name>
+```
+
+The unit tests require cross-env to be installed:
+```
+npm install cross-env
```

Comment:
This step is not needed, it should be 'npm install' on its own

Reply:
Fixed